src/services/outliningElementsCollector.ts

```diff
@@ -34,7 +34,7 @@ namespace ts.OutliningElementsCollector {
             if (depthRemaining === 0) return;
             cancellationToken.throwIfCancellationRequested();
 
-            if (isDeclaration(n) || isVariableStatement(n) || isReturnStatement(n) || n.kind === SyntaxKind.EndOfFileToken) {
+            if (isDeclaration(n) || isVariableStatement(n) || isReturnStatement(n) || isCallOrNewExpression(n) || n.kind === SyntaxKind.EndOfFileToken) {

```

Comment:
If this is the fix, I don't understand why the bug only reproduced when the comment text had no leading spaces on its line. It feels like there must be something else at play as well?

Reply:
Actually, I could not get an outline with space in the comment

<img width="218" alt="Screenshot 2021-11-04 at 21 35 41" src="https://user-images.githubusercontent.com/509265/140408119-8a25a33a-c13a-45f7-872b-b380ebcb8326.png">
<img width="198" alt="Screenshot 2021-11-04 at 21 35 48" src="https://user-images.githubusercontent.com/509265/140408124-2515b145-d6b7-4b65-be31-cd40141bd147.png">
<img width="198" alt="Screenshot 2021-11-04 at 21 35 59" src="https://user-images.githubusercontent.com/509265/140408126-c2eb6b5e-03a6-4938-b7f0-8f64c925cde8.png">

The comment associated with the call expression was not included in the outline spans.

Reply:
I see—most be a difference with Monaco, because that’s the behavior I was seeing in the Playground.

Reply:
@andrewbranch Oke, thanks for the review 👍🏻